run_node.sh

```diff
@@ -31,19 +38,38 @@ get_version () {
     # The version that is returned by the extrinsic looks like: 0.11.4-ae34eb4213
     # so we take the hash part and use it to identify docker images and commits in the repo.
 
-    if  [ -n "${MAINNET}" ]
+    if [ -n "${VERSION}" ]
     then
-        VERSION_RPC_URL="https://rpc.azero.dev"
-    else
-        VERSION_RPC_URL="https://rpc.test.azero.dev"
+        echo ""
+        echo -e "Version manually set to ${BGREEN}${VERSION}${NC}."
```

Comment:
Can we please use those wrappers? https://github.com/Cardinal-Cryptography/aleph-node/blob/main/scripts/common.sh#L123-L134

Colors are a great feature to the non-CI scripts as runner is, though as it is now code os a bit obfuscated.

Reply:
Yeah, so I tried to alleviate this to some extent, using these wrappers where I could, thanks! Some of the colors are only set for a part of the message so for now I'll leave them as they are if that's okay -- IMO small price to pay for these beautiful colors! :D